### PR TITLE
[BPKR-179]: Fixed close button margins issue in RTL

### PR DESCRIPTION
**Pull request by olliecurtis**
Remember to include the following changes:

- [x] `UNRELEASED.md`
- [ ] `README.md`
- [ ] Tests
- [ ] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

### CODE REVIEW

File: packages/bpk-component-modal/src/BpkModalInner.scss [hunk 1]
```diff
@@ -95,6 +95,7 @@
     @include bpk-rtl {
```
Comment (k0nserv) [inline packages/bpk-component-modal/src/BpkModalInner.scss:95]: Might I suggest adopting `bpk-margin-trailing` from `bpk-mixins`?

Comment (olliecurtis) [reply packages/bpk-component-modal/src/BpkModalInner.scss:95]: Sure!

File: packages/bpk-mixins/src/mixins/_margins.scss [hunk 2]
```diff
@@ -19,7 +19,7 @@
 /// RTL aware leading margin
 ///
 /// @param {string} $margin The desired leading margin.
-/// @param {boolean} $resetOppositeMargin [false] Whether to reset the opposite margin to 0.
+/// @param {boolean} $resetOppositeMargin [true] Whether to reset the opposite margin to 0.
```
Comment (olliecurtis) [inline packages/bpk-mixins/src/mixins/_margins.scss:22]: Corrected the docs here as the default value state was incorrect

File: packages/bpk-mixins/src/mixins/_margins.scss [hunk 3]
```diff
@@ -46,7 +46,7 @@
 /// RTL aware trailing margin
 ///
 /// @param {string} $margin The desired trailing margin.
-/// @param {boolean} $resetOppositeMargin [false] Whether to reset the opposite margin to 0.
+/// @param {boolean} $resetOppositeMargin [true] Whether to reset the opposite margin to 0.
```
Comment (olliecurtis) [inline packages/bpk-mixins/src/mixins/_margins.scss:49]: Corrected the docs here as the default value state was incorrect

File: packages/bpk-mixins/src/mixins/_margins.scss (hunk 2 above)
Comment (k0nserv) [reply packages/bpk-mixins/src/mixins/_margins.scss:22]: We should probably mention this in the changelog

**Review by k0nserv — APPROVED** [review]

**Pull request merged**